spec/inject_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'inject'
+
+describe Array do
+
+  xit 'can add' do
+    expect([1, 2, 3, 4].inject { |memo, element| memo + element }).to eq 10
+    expect([1, 2, 3, 4].my_inject { |memo, element| memo + element }).to eq 10
+  end
+
+  xit 'can add with a starting point' do
+    expect([1, 2, 3, 4].inject(10) { |memo, element| memo + element }).to eq 20
+    expect([1, 2, 3, 4].my_inject(10) { |memo, element| memo + element }).to eq 20
+  end
+
+  xit 'can multiply' do
+    expect([2, 2, 3].inject { |memo, element| memo * element }).to eq 12
+    expect([2, 2, 3].my_inject { |memo, element| memo * element }).to eq 12
+  end
+
+  xit 'can multiply with a starting point' do
+    expect([1, 2, 3, 4].inject(10) { |memo, element| memo * element }).to eq 240
+    expect([1, 2, 3, 4].my_inject(10) { |memo, element| memo * element }).to eq 240
+  end
+
+  xit 'can add with a symbol' do
+    expect([1, 2, 3, 4].inject(:+)).to eq 10
+    expect([1, 2, 3, 4].my_inject(:+)).to eq 10
+  end
+
+  xit 'can add with a symbol and a starting point' do
+    expect([1, 2, 3, 4].inject(10, :+)).to eq 20
+    expect([1, 2, 3, 4].my_inject(10, :+)).to eq 20
+  end
+
+  xit 'can find the longest string' do
+    expect(['one', 'two', 'three', 'four', 'five'].my_inject do |memo, number|
+       if memo.length > number.length
+         memo
+       else
+         number
+       end
+     end).to eq "three"
```

Comment:
`end` at 42, 5 is not aligned with `['one', 'two', 'three', 'four', 'five'].my_inject do |memo, number|` at 36, 11 or `expect(['one', 'two', 'three', 'four', 'five'].my_inject do |memo, number|` at 36, 4